### PR TITLE
Main header was created

### PR DESCRIPTION
**Pull request by javascriptmruser2**
`<h1>Privet</h1>`

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    <div class="main-header">
```
Comment (universezzz) [inline index.html]: Sry, but why do you not use `<header> ` instead of `<div>`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        <div class="main-nav">
```
Comment (universezzz) [inline index.html]: Sry, but why do you not use `<nav>` instead of `<div>`

File: index.html [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+            <div class="navigation-list"></div>
```
Comment (universezzz) [inline index.html]: Sry, but why do you not use `<ul>` instead of `<div>`

File: index.html (hunk 2 above)
Comment (javascriptmruser2) [reply index.html]: Resolved.

File: index.html (hunk 1 above)
Comment (javascriptmruser2) [reply index.html]: Resolved.

File: index.html (hunk 3 above)
Comment (javascriptmruser2) [reply index.html]: Resolved.

**Pull request merged**